flink/templates/sealedsecretenvs.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.secret.sealedSecrets.sealedSecretEnvs }}
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  annotations:
+    sealedsecrets.bitnami.com/namespace-wide: "true"
```

Comment:
Could you please make it customizable from values.yaml?
according to README on https://github.com/bitnami-labs/sealed-secrets this annotation will be changed in next realese

Reply:
Ye, you are right. Good catch!